charts/generic/k8-wavefront-collector/experiment.yaml

```diff
@@ -0,0 +1,58 @@
+# Generic Chaos experiment for Application team, who want to participate in Game Day
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Deletes a pod belonging to a deployment/statefulset/daemonset
+kind: ChaosExperiment
+metadata:
+  name: k8-pod-delete
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      - apiGroups: ["","apps","batch"]
+        resources: ["jobs","deployments","daemonsets"]
+        verbs: ["create","list","get","patch","delete"]
+      - apiGroups: ["","litmuschaos.io"]
+        resources: ["pods","configmaps","events","services","chaosengines","chaosexperiments","chaosresults","deployments","jobs"]
+        verbs: ["get","create","update","patch","delete","list"] 
+      - apiGroups: [""]
+        resources: ["nodes"]
+        verbs : ["get","list"]
+    labels:
+      name: k8-pod-delete    
+      app.kubernetes.io/part-of: litmus
+    image: "docker.artifactory.a.intuit.com/oicp/standard/external/chaostoolkit:latest"
```

Comment:
the docker image should be updated to litmuschaos/chaostoolkit:latest

Reply:
fixed